### PR TITLE
Bestpractice

### PR DESCRIPTION
**Pull request by mkhalil8**
I have made two files with the same changes, kindly ignore the second one.

### CODE REVIEW

**Comment by ntolley** [comment]
@mkhalil8 thanks for starting the pull request! I'm a little confused by the file that's committed here, is it the output after building the html?

In any case a good example of editing existing documentation is #249. You can see that the "diff" when you click on "files changed" only includes desired updates in green, while a large amount remains untouched.

There's unfortunately a bit of a learning curve to merging PR's, but all the others are infinitely easier after the first one. Please let us if you want to set up a meeting about contributing, and the subject of this PR in particular!

File: doc/auto_examples/workflows/plot_simulate_evoked.rst [hunk 1]
```diff
@@ -0,0 +1,466 @@
...
+
+    <Figure size 640x480 with 1 Axes>
+
+
+
+.. GENERATED FROM PYTHON SOURCE LINES 155-160
+.. warning::
+   Always look at dipoles in conjunction with raster plots and spike histogram to avoid misinterpretation.
+
+   Run multiple trials for your simulation to get an average of different drives seeds before drawing conclusions.
```
Comment (jasmainak) [inline doc/auto_examples/workflows/plot_simulate_evoked.rst:420]: You should put this in the python script as comments. This file you edited is auto generated ...

**Comment by mkhalil8** [comment]
> @mkhalil8 thanks for starting the pull request! I'm a little confused by the file that's committed here, is it the output after building the html?
> 
> In any case a good example of editing existing documentation is #249. You can see that the "diff" when you click on "files changed" only includes desired updates in green, while a large amount remains untouched.
> 
> There's unfortunately a bit of a learning curve to merging PR's, but all the others are infinitely easier after the first one. Please let us if you want to set up a meeting about contributing, and the subject of this PR in particular!

@ntolley yes that would be great if we can have a meeting, I believe it will not be the only time to try and contribute so help is needed :)
 I have read a bit about Sphinx and my understanding was that the .rst files are the ones that cause changes to the html. I updated the plot_simulate_evoked.rst and the html page followed so I thought I am on the right track :)

File: doc/auto_examples/workflows/plot_simulate_evoked.rst (hunk 1 above)
Comment (mkhalil8) [reply doc/auto_examples/workflows/plot_simulate_evoked.rst:420]: > You should put this in the python script as comments. This file you edited is auto generated ...

@jasmainak  👍  I thought adjusting the .rst file is how you do it :)
will update the python script then make a new pull request.

**Comment by jasmainak** [comment]
would be great if you and @ntolley can meet. But ultimately you'll learn from the mistakes :)

sphinx-gallery is what we use: https://sphinx-gallery.github.io/stable/index.html. It's a layer on top of sphinx that converts the python files into rst that is finally built into html. The automatically generated rst should not be touched. You can check the rendering by doing:

```sh
$ cd doc/
$ make html-noplot
```

or `$ make html` if you want to run the examples and see the outputs as well

**Comment by jasmainak** [comment]
@mkhalil8 any luck here?

**Comment by mkhalil8** [comment]
@jasmainak I had couple of trials but something is off and the HTML is not updated. I have a meeting with Nick on Friday morning to sort it out.

**Pull request closed without merging**